app/lib/active_model/v08/serializer.rb

```diff
@@ -8,6 +8,7 @@ class Serializer < ActiveModel::Serializer
       # AMS 0.8 would delegate method calls from within the serializer to the
       # object.
       def method_missing(*args)
+        super
```

Comment:
Why have you added the 'super' in here, but not in other places? e.g. DerivedVariable

Reply:
Because it broke everything when done in the other places(!)